cmd/csi_driver/main.go

```diff
@@ -56,7 +56,8 @@ var (
 	retryIntervalStart             = flag.Duration("retry-interval-start", time.Second, "Initial retry interval for a failed PV processing operation. It doubles with each failure, up to retry-interval-max.")
 	retryIntervalMax               = flag.Duration("retry-interval-max", 5*time.Minute, "Maximum retry interval for a failed PV processing operation.")
 	fuseSocketDir                  = flag.String("fuse-socket-dir", "/sockets", "FUSE socket directory.")
-	metricsEndpoint                = flag.String("metrics-endpoint", "", "The TCP network address where the Prometheus metrics endpoint will listen (example: `:8080`). The default is empty string, which means that the metrics endpoint is disabled.")
+	metricsEndpoint                = flag.String("metrics-endpoint", "", "(deprecated) The TCP network address where the Prometheus metrics endpoint will listen (example: `:8080`). The default is empty string, which means that the metrics endpoint is disabled. If set, `--http-endpoint` cannot be set")
+	httpEndpoint                   = flag.String("http-endpoint", "", "The TCP network address where the HTTP server for diagnostics, including CSI driver health check and metrics. The default is empty string, which means the server is disabled. If set, `--metrics-endpoint` cannot be explicitly set.")
```

Comment:
Should we add example here as well? Similar to how we add example for metrics-endpoint?